files/bootstrap-fileinput/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Kartik Visweswaran"
+github = "https://github.com/kartik-v"
```

Comment:
This is the project's repo not the author's
So: `https://github.com/kartik-v/bootstrap-fileinput`